packages/spaces-image/SpacesImage.js

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useEffectAsync } from '@availity/hooks';
+import { avSlotMachineApi } from '@availity/api-axios';
+import get from 'lodash.get';
+
+const spaceIDQuery = `
+query($id: ID!){
+  space(id: $id){
+    images{
+      name
+      value
+    }
+  }
+}
+`;
+
+const payerIDQuery = `
+query($payerIDs: [String!], $types: [String!]){
+  spaces(payerIDs: $payerIDs, types: $types){
+    spaces{
+      images{
+        name
+        value
+      }
+    }
+  }
+}
+`;
+
+const fetchImage = async (query, variables, path, clientId, imageType) => {
+  try {
+    const {
+      data: { data },
+    } = await avSlotMachineApi.create(
+      {
+        query,
+        variables,
+      },
+      { headers: { 'X-Client-ID': clientId } }
```

Comment:
I can see in the future where we expose some function say from `sdk-js` where we auto apply the clientId to all requests. In that case it would be nice to not require the `clientId` as a prop for some cases. If the clientId is `undefined` here, as long as it doesn't overwrite a global one that was placed before it this looks good.

Otherwise we need to add a conditional check to add that `key/value` to the request.